docker/python-computation/Dockerfile

```
@@ -1,47 +1,41 @@
-# Install PIP requirements
-ADD requirements.txt /tmp/
-RUN pip3 install -r /tmp/requirements.txt
+# ---- Install the swh package (editable) ----
+# The repo is mounted at /opt/social_warehouse/, so we install from there.
+# During build, we copy just pyproject.toml to cache the dependency layer.
+COPY requirements.txt /tmp/
+RUN pip install -r /tmp/requirements.txt
 
 WORKDIR /opt/social_warehouse
 
 ADD entrypoint.sh /usr/local/bin/
```

Hadolint (2.14.0)

[error] Use COPY instead of ADD for files and folders

(DL3020)